bundles/org.openhab.binding.shelly/src/main/java/org/openhab/binding/shelly/internal/handler/ShellyBaseHandler.java

```diff
@@ -808,7 +853,7 @@ protected void initializeThingConfig() {
 
         config = getConfigAs(ShellyThingConfiguration.class);
         if (config.deviceIp.isEmpty()) {
-            logger.warn("{}: IP address for the device must not be empty", thingName); // may not set in .things file
+            logger.info("{}: IP address for the device must not be empty", thingName); // may not set in .things file
```

Comment:
This is checked by the framework, if you defined the parameter in the XML file sufficiently. So, this could be removed entirely.